### PR TITLE
MX-15032: MaxVotingDelayPeriodInEpochs

### PR DESCRIPTION
**Pull request by dragos-rebegea**
## Reasoning behind the pull request
- Prevent propose voting too far into the future

### CODE REVIEW

File: config/systemSmartContractsConfig.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ type ESDTSystemSCConfig struct {
 }
 
 // GovernanceSystemSCConfigV1 holds the initial set of values that were used to initialise the
-//  governance system smart contract at genesis time
+//
```
Comment (iulianpascalau) [inline config/systemSmartContractsConfig.go:35]: indenting went wrong in this file

File: config/systemSmartContractsConfig.go [hunk 2]
```diff
@@ -42,7 +43,8 @@ type GovernanceSystemSCConfigV1 struct {
 }
 
 // GovernanceSystemSCConfigActive defines the set of configuration values used by the governance
-//  system smart contract once it activates
+//
+//	system smart contract once it activates
```
Comment (sasurobert) [inline config/systemSmartContractsConfig.go:47]: wrong indentation

File: config/systemSmartContractsConfig.go (hunk 1 above)
Comment (dragos-rebegea) [reply config/systemSmartContractsConfig.go:35]: fixed

File: config/systemSmartContractsConfig.go (hunk 2 above)
Comment (dragos-rebegea) [reply config/systemSmartContractsConfig.go:47]: fixed

**Review by iulianpascalau — APPROVED** [review]

File: config/systemSmartContractsConfig.go (hunk 2 above)
Comment (dragos-rebegea) [reply config/systemSmartContractsConfig.go]: fixed

**Review by sasurobert — APPROVED** [review]

**Pull request merged**